src/main/java/edu/wpi/grip/core/Step.java

```diff
@@ -12,6 +13,7 @@
  * A step is an instance of an operation in a pipeline.  A step contains a list of input and output sockets, and it
  * runs the operation whenever one of the input sockets changes.
  */
+@XStreamAlias(value = "grip:Step")
```

Comment:
You can access static final variables in annotations. Don't know if you want to do that here.